gateway-cli/src/main.rs

```diff
@@ -478,8 +478,8 @@ async fn main() -> Result<()> {
             } else {
                 // clap guarantees these are not `None` when `clear` is false.
                 let update_id = update_id.unwrap();
-                let host_phase_2 = host_phase_2.unwrap().to_string();
-                let control_plane = control_plane.unwrap().to_string();
+                let host_phase_2 = host_phase_2.unwrap();
+                let control_plane = control_plane.unwrap();
```

Comment:
Do you happen to know why we import `omicron_common::update::ArtifactHash` for these rather than `gateway_client::types::ArtifactHash`?